Aquality.Selenium.Core/tests/Aquality.Selenium.Core.Tests/Resources/settings.custom.json

```diff
@@ -0,0 +1,15 @@
+{
+  "timeouts": {
+    "timeoutImplicit": 0,
+    "timeoutCondition": 30,   
+    "timeoutPollingInterval": 300,
+    "timeoutCommand": 60
+  },
+  "retry": {
+    "number": 2,
+    "pollingInterval": 300
+  },
+  "logger": {
+    "language": "be"
```

Comment:
maybe 'by' to correspond to https://countrycode.org/
'be' means Belgium

Reply:
BE is a country-code of Belgium, not the language code, since there is no such thing as "belgium language" in ISO standard.
For example, in en-UK locale "en" stands for English, and UK stands for United Kingdom. BUT in uk-UA locale code, "uk" stands for Ukrainian, and UA stands for Ukraine.
Please look at official ISO standard here: https://www.loc.gov/standards/iso639-2/php/code_list.php or everywhere else.

P.S. be stands for Belarusian language, and BY stands for Byelorussia with was the official name of Byelorussian SSR delegation at United Nation's founding at 1945.
P.P.S. We do have "en" in library not for England but for English language.